src/pages/preferences/preferencesTabs/cliTab/VerifyCard.tsx

```diff
@@ -21,11 +21,7 @@ export const VerifyCard = () => {
   );
 
   const latestRevision = get(data, "clientConfig.latestRevision", "");
-  const verificationCode = `
-"[evergreen] ${format(new Date(), "yyyy/MM/dd hh:mm:ss")} [p=notice]:
-[message='Binary is already up to date - not updating.'
-revision='${latestRevision}']"
-  `;
+  const verificationCode = "Binary is already up to date - not updating.";
```

Comment:
I think it still prints the revision even with the plain logger doesn't it? Otherwise you should remove the unused constant in the line above

Reply:
Fixed